catalog/static/.prettierrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "trailingComma": "es5"
```

Comment:
won't this file be added to the build?

Reply:
Yes
I added `ignore` rule to CopyWebpackPlugin config